pkg/test/resourcefixture/testdata/basic/netapp/v1alpha1/netappbackupvault/netappbackupvault-basic/_http.log

```diff
@@ -231,58 +136,28 @@ User-Agent: kcc/${kccVersion} (+https://github.com/GoogleCloudPlatform/k8s-confi
 X-Goog-Request-Params: name=projects%2F${projectId}%2Flocations%2Fus-west2%2FbackupVaults%2Fnetappbackupvault-basic-${uniqueId}
 
 200 OK
-Content-Type: application/json; charset=UTF-8
-Server: ESF
-Vary: Origin
-Vary: X-Origin
-Vary: Referer
-X-Content-Type-Options: nosniff
-X-Frame-Options: SAMEORIGIN
-X-Xss-Protection: 0
-
-{
-  "done": false,
-  "metadata": {
-    "@type": "type.googleapis.com/google.cloud.netapp.v1.OperationMetadata",
-    "apiVersion": "v1",
-    "createTime": "2024-04-01T12:34:56.123456Z",
-    "requestedCancellation": false,
-    "target": "projects/${projectId}/locations/us-west2/backupVaults/netappbackupvault-basic-${uniqueId}",
-    "verb": "delete"
-  },
-  "name": "projects/${projectId}/locations/us-west2/operations/${operationID}"
-}
-
----
-
-GET https://netapp.googleapis.com/v1/projects/${projectId}/locations/us-west2/operations/${operationID}
 Content-Type: application/json
-User-Agent: kcc/${kccVersion} (+https://github.com/GoogleCloudPlatform/k8s-config-connector) kcc/controller-manager/${kccVersion}
-X-Goog-Request-Params: name=projects%2F${projectId}%2Flocations%2Fus-west2%2Foperations%2F${operationID}
-
-200 OK
-Content-Type: application/json; charset=UTF-8
-Server: ESF
-Vary: Origin
-Vary: X-Origin
-Vary: Referer
-X-Content-Type-Options: nosniff
-X-Frame-Options: SAMEORIGIN
-X-Xss-Protection: 0
+Grpc-Metadata-Content-Type: application/grpc
 
 {
   "done": true,
   "metadata": {
     "@type": "type.googleapis.com/google.cloud.netapp.v1.OperationMetadata",
-    "apiVersion": "v1",
-    "createTime": "2024-04-01T12:34:56.123456Z",
-    "endTime": "2024-04-01T12:34:56.123456Z",
+    "apiVersion": "",
+    "createTime": null,
+    "endTime": null,
     "requestedCancellation": false,
-    "target": "projects/${projectId}/locations/us-west2/backupVaults/netappbackupvault-basic-${uniqueId}",
-    "verb": "delete"
+    "statusMessage": "",
+    "target": "",
+    "verb": ""
```

Comment:
Any reason we are not trying to match these LRO metadata?

Reply:
I didn't know we wanted to do that 😛

Reply:
Yeah, I think we should aim to minimize the differences, as long as it doesn't require excessive effort.